manifests/init.pp

```diff
@@ -147,6 +152,7 @@
   $package_provider   = $jenkins::params::package_provider,
   $service_enable     = $jenkins::params::service_enable,
   $service_ensure     = $jenkins::params::service_ensure,
+  $service_flags      = undef,
```

Comment:
If this is required, does it need a default value for openbsd in `jenkins::params`?